SqlDatabases/DatabaseRequest.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Linq2Azure.SqlDatabases
+{
+    public class DatabaseRequest 
```

Comment:
Convention in L2A is to pass an instance of the primary representation to a create method on the relevant parent. For instance on Subscription:

public Task CreateCloudServiceAsync(CloudService service) { return service.CreateAsync(this); }